### ISSUE TITLE
Bootloader active

### ISSUE THREAD

**Issue by lucasimons** [syssi/esphome-evse-wallbox#19]
Hi, i have a evse din since 2022 but last month i had some issue, i think that in strange situation enable the bootloader 

![image](https://github.com/syssi/esphome-evse-wallbox/assets/67467751/121d26fa-18c3-4f00-9f84-3aba22bd7aff)

I purchased another evse din board and after a week same thing, green LED and no communication. I tried to contact the manufacturer but he didn't give me any suggestions...

![IMG_20240106_122501.jpg](https://github.com/syssi/esphome-evse-wallbox/assets/67467751/9e8475dd-3d79-4ca8-b2c5-1f40406e5142)

**Comment by lucasimons** [comment]
Hi, I finally managed to solve the problem of the active bootloader, the seller actually says that it deactivates after 5 minutes but in reality it didn't work for me.
So I had to flash the firmware to the PIC16F1825 using a pickit 3

![image](https://github.com/syssi/esphome-evse-wallbox/assets/67467751/3a35a0db-8585-4e69-8e4c-e1231775cccf)
the next days i prepare all schematics to flash it

**Comment by syssi** [comment]
I must admit I don't understand this implementation can cause the issue or it's just a strange behaviour of the device sometimes? How do you like to proceed here? I would like to close the issue some day to not confuse/disturb other users.

**Comment by footswitch** [comment]
> I must admit I don't understand this implementation can cause the issue or it's just a strange behaviour of the device sometimes? How do you like to proceed here? I would like to close the issue some day to not confuse/disturb other users.

I agree that this should be closed, since there is no demonstration/steps to reproduce the problem.
